src/main/java/com/teragrep/cfe_39/consumers/kafka/DatabaseOutput.java

```diff
@@ -0,0 +1,394 @@
+import com.teragrep.cfe_39.avro.SyslogRecord;
+import com.teragrep.cfe_39.consumers.kafka.queue.WritableQueue;
+import com.teragrep.cfe_39.metrics.topic.TopicCounter;
+import com.teragrep.cfe_39.metrics.DurationStatistics;
+import com.teragrep.rlo_06.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.time.Instant;
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.function.Consumer;
+
+import java.nio.ByteBuffer;
+
+//  The kafka stream should first be deserialized using rlo_06 and then serialized again using avro and stored in HDFS.
+//  The target where the record is stored in HDFS is based on the topic, partition and offset. ie. topic_name/0.123456 where offset is 123456
+
+// The mock consumer is activated for testing using the configuration file: readerKafkaProperties.getProperty("useMockKafkaConsumer", "false")
+
+public class DatabaseOutput implements Consumer<List<RecordOffset>> {
+    private static final Logger LOGGER = LoggerFactory.getLogger(DatabaseOutput.class);
+    private final RFC5424Frame rfc5424Frame = new RFC5424Frame(false);
+
+    private final String table;
+
+
+    private final DurationStatistics durationStatistics;
+    private final TopicCounter topicCounter;
+
+    private long lastTimeCalled = Instant.now().toEpochMilli();
+
+    public static final String ANSI_RESET = "\u001B[0m";
+    public static final String ANSI_GREEN = "\u001B[32m";
+    public static final String ANSI_BLUE = "\u001B[34m";
+    private SyslogAvroWriter syslogAvroWriter;
+    //    private final long minimumFreeSpace; // TODO: Check if needs to be implemented to the code.
+    private final long maximumFileSize;
+    private final WritableQueue writableQueue;
+    private final ByteBuffer sourceConcatenationBuffer;
+    private final SDVector teragrepStreamName;
+    private final SDVector teragrepDirectory;
+    private final SDVector eventNodeSourceSource;
+    private final SDVector eventNodeRelaySource;
+    private final SDVector eventNodeSourceSourceModule;
+    private final SDVector eventNodeRelaySourceModule;
+    private final SDVector eventNodeSourceHostname;
+    private final SDVector eventNodeRelayHostname;
+    private final SDVector originHostname;
+    private File syslogFile;
+    private final Config config;
+    //    private long approximatedSize; // FIXME: Not working properly when flush() is not used after append in the AVRO-file.
+    private long epochMicros_last;
+
+    DatabaseOutput(
+            Config config,
+            String table,
+            DurationStatistics durationStatistics,
+            TopicCounter topicCounter
+    ) {
+        this.config = config;
+        this.table = table;
+        this.durationStatistics = durationStatistics;
+        this.topicCounter = topicCounter;
+//        this.minimumFreeSpace = 32000000; // TODO: CHECK RIGHT VALUE FOR minimumFreeSpace
```

Comment:
Don't comment out code

Reply:
Solved in commit 022832c8b423efc7466b7aead56cc317a45a0002